packages/server/src/queries/complex/orderbooks/active-orders.ts

```diff
@@ -16,6 +17,46 @@ import type { MappedLimitOrder, OrderStatus } from "./types";
 
 const activeOrdersCache = new LRUCache<string, CacheEntry>(DEFAULT_LRU_OPTIONS);
 
+export function getOrderbookActiveOrdersSQS({
+  userOsmoAddress,
+  assetList,
+}: {
+  userOsmoAddress: string;
+  assetList: AssetList[];
+}) {
+  return cachified({
+    cache: activeOrdersCache,
+    key: `orderbookActiveOrders-sqs-${userOsmoAddress}`,
+    ttl: 5000, // 5 seconds
```

Comment:
I have a personal rule of never caching user account queries just to be safe. But, it's a bit extreme and I think it's fine with low ttl queries

Reply:
I think we want to be strict with this query in terms of refetch/cache times and loosen it once we're comfortable it won't cause any issues with SQS